### PR TITLE
Feat(OnboardingRegistration): Creates registration page

### PR DESCRIPTION
**Pull request by vakodrazan**
Hello Team,

The registration functionalities are ready for review. Please have a look at it whenever you are free. Feel free to leave some comments on things that need to improve.

Thanks in advance,

[Task link](https://app.clickup.com/t/41k8d6)
Refs: 41k8d6

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React from 'react'
 import './App.css'
 import { HomePage } from './Pages/HomePage'
-import { RoleSelectOnboard } from './components/RoleSelectionOnboard/RoleSelectionOnBoard'
+// import { RoleSelectOnboard } from './components/RoleSelectionOnboard/RoleSelectionOnBoard'
```
Comment (andrzej-onja) [inline src/App.tsx]: remove unused

File: src/Pages/OnboardingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import { loggeOut } from '../components/HeaderNavLink/HeaderNavLink'
+
+import footerLinks from '../utils/FooterLinks'
+import { useSelector } from 'react-redux'
+import { userSelector } from '../redux/slices/userSlice'
+
+const RegisteredSuccessFully = () => (
+  <div>
+    <h1>Registered Succesfully</h1>
+  </div>
```
Comment (andrzej-onja) [inline src/Pages/OnboardingPage.tsx]: This doesn't match the design

File: src/components/RoleSelectionOnboard/RoleSelectionOnBoard.tsx [hunk 3]
```diff
@@ -17,64 +11,47 @@ import userIconHovered from '../../stories/assets/userNotHovered.svg'
...
-import footerLinks from '../../utils/FooterLinks'
+export interface HandlerProp {
+  setRoleSelected?: any
+}
 
-export const RoleSelectOnboard = () => {
+export const RoleSelectOnboard: FC<HandlerProp> = ({ setRoleSelected }) => {
   /* eslint-disable @typescript-eslint/no-unused-vars */
-  const [roleSelected, setRoleSelected] = useState('')
+  // const [roleSelected, setRoleSelected] = useState('')
```
Comment (andrzej-onja) [inline src/components/RoleSelectionOnboard/RoleSelectionOnBoard.tsx:23]: remove if unused

File: src/components/RoleSelectionOnboard/RoleSelectionOnBoardiStyle.tsx [hunk 4]
```diff
@@ -16,9 +16,10 @@ export const TopContainerStyles = styled.div`
   `}
 `
 export const MainContainer = styled.div`
-  position: absolute;
+  /* position: absolute;
   transform: translateX(-50%);
-  left: 50%;
+  left: 50%; */
```
Comment (andrzej-onja) [inline src/components/RoleSelectionOnboard/RoleSelectionOnBoardiStyle.tsx]: remove commented out code

File: src/Pages/LoginPage.tsx [hunk 5]
```diff
@@ -74,65 +69,60 @@ export const LoginPage: React.FC<LoginPageProps> = ({
...
-            </RegistrationOnboardingContent>
-          </RegistrationOnboardingContainer>
+                <Button
+                  type='button'
+                  label='Login with Google'
+                  buttonIcon={ButtonIcon}
+                />
+              </RegistrationButtonContainer>
+            </RegistrationForm>
+          </RegistrationOnboardingContent>
```
Comment (andrzej-onja) [inline src/Pages/LoginPage.tsx:125]: Why registration is in Login page ?

Comment (andrzej-onja) [reply src/Pages/LoginPage.tsx:125]: It should be renamed to login. I see its not part of your PR so live it as it is just add todo at the top that it needs to be renamed please. Approving in advance

**Review by andrzej-onja — APPROVED** [review]

**Review by tsipoy — APPROVED** [review]

**Review by bakozetra — APPROVED** [review]

**Pull request merged**